src/console.rs

```diff
@@ -57,6 +60,74 @@ impl ConsoleLog {
     }
 }
 
+/// This is useful, but not implemented well. This needs to be either reworked and reimplemented, or utilised better.
+/// TODO: fix this
```

Comment:
Yeah I don't love it either. Hopefully we can find a nicer way to do this at some point.

Reply:
remove the unnessescary serialise fn